04_numeronym/brvishnu92/main_test.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+//main() test
+func TestMainOutput(t *testing.T) {
```

Comment:
I don't see the value in this comment. The name of the test function is documentation in itself for what the function does.